### DISCUSSION TITLE
Allowing specific items pass by an Ignore command

### DISCUSSION THREAD

**Discussion by Soiltasun** [hluk/CopyQ#2529] [Q&A]
Ignore for password managers works flawlessly, but a similar window from their web support page 'Password Support' is also ignored. Is it possible to put a hard stop/non-asterixis after ignore 'password' so that 'password support' does permit copy/paste functionality?

**Comment by hluk** [comment]
If you are talking about the "Window" field in "Advanced" tab in command settings (in Command dialog), it accepts regular expression, so to match suffix you can use `password$`. Prefix would be `^password` and full match `^password$`.

**Reply by Soiltasun** [reply]
Thank you for the reply; tried fruitlessly to get a correct regex expression. Fortunately the window currently has 'Contact Us' ahead of 'Password Support' so a top level command using 'copyq enable' with 'Contact Us' in the window box allows it to be copied with the subsequent command ignoring 'Password' window titles.

Have to believe there is a way to combine these two commands via regex (ignoring 'password' titles but allowing them when 'contact us' or some other term is also present) but couldn't suss it out through the many combinations tried using regex101

**Reply by hluk** [reply]
You can use more complex logic in "Filter" field. Something like:

```js
copyq:
const title = str(data(mimeWindowTitle));
if (/Contact Us/.test(title))
    fail();
if (!/password/i.test(title))
    fail();
// the command will be applied to any other window title
// i.e. not containing "Contact Us" and containing "password"
```
